docs/src/transforms.md

```diff
@@ -480,45 +480,45 @@ This transform will drop messages that match the specified filter. You can eithe
     # DenyList: [Write, ReadWrite, SchemaChange, PubSubMessage]
 ```
 
-### RedisCache
+### ValkeyCache
 
-This transform will attempt to cache values for a given primary key in a Redis hash set. It is a primarily implemented as a read behind cache. It currently expects an SQL based AST to figure out what to cache (e.g. CQL, PGSQL) and updates to the cache and the backing datastore are performed sequentially.
+This transform will attempt to cache values for a given primary key in a Valkey hash set. It is a primarily implemented as a read behind cache. It currently expects an SQL based AST to figure out what to cache (e.g. CQL, PGSQL) and updates to the cache and the backing datastore are performed sequentially.
 
 ```yaml
-- RedisCache:
+- ValkeyCache:
     caching_schema:
       test:
         partition_key: [test]
         range_key: [test]
     chain:
-      # The chain can contain anything but must end in a Redis sink
-      - RedisSinkSingle:
+      # The chain can contain anything but must end in a Valkey sink
+      - ValkeySinkSingle:
           # The IP address and port of the upstream redis node/service.
```

Comment:
There are still some `redis` in the comments like this one.

Reply:
good catch, fixed